src/gobapi/dump/sql.py

```diff
@@ -0,0 +1,96 @@
+"""
+Dump GOB
+
+Dumps of catalog collections in sql format
+"""
+from gobcore.model import GOBModel
+
+from gobapi.dump.config import DELIMITER_CHAR
+from gobapi.dump.config import UNIQUE_ID, REFERENCE_TYPES, REFERENCE_FIELDS
+from gobapi.dump.config import SCHEMA, SQL_TYPE_CONVERSIONS
+
+from gobapi.dump.config import get_field_specifications, joined_names
+
+
+def _create_schema(name):
+    """
+    Returns a SQL statement to create a schema with the given name
+
+    :param name:
+    :return:
+    """
+    return f"""
+-- DROP SCHEMA {name} CASCADE;
+CREATE SCHEMA IF NOT EXISTS {name};
+"""
+
+
+def _create_table(schema, table, specs):
+    """
+    Returns a SQL statement to create a table in a schema
+    The table fields are constructed from the specs
+
+    :param schema:
+    :param table:
+    :param specs:
+    :return:
+    """
+    fields = []
+    for field_name, field_spec in specs.items():
+        if field_spec['type'] in REFERENCE_TYPES:
+            for reference_field in REFERENCE_FIELDS:
+                fields.append(f"{joined_names(field_name, reference_field)} {SQL_TYPE_CONVERSIONS['GOB.String']}")
+        else:
+            fields.append(f"{field_name} {SQL_TYPE_CONVERSIONS[field_spec['type']]}")
+    fields = ",\n  ".join(fields)
+    return f"""
+DROP TABLE IF EXISTS {schema}.{table} CASCADE;
```

Comment:
I would quote all SQL identifiers (schema, table, column names) to prevent weird errors with SQL keywords accidentally being used in identifiers.

Also; note that quotation marks may differ per database type.